### PR TITLE
HOCS-2226: Update govuk-frontend package from 2.9.0 to 3.10.0

### PR DESCRIPTION
**Pull request by parth-ho**
note: The guidance I used to replace the old colours is here: https://frontend.design-system.service.gov.uk/migrating-from-legacy-products/#2-turn-off-compatibility-mode

### CODE REVIEW

File: src/styles/app.scss [hunk 1]
```diff
@@ -57,7 +57,7 @@ $govuk-font-family-tabular: 'helvetica neue',Arial, Helvetica, sans-serif !defau
   padding: 0 govuk-spacing(1);
   overflow-x: auto;
   color: govuk-colour("pink");
-  background-color: govuk-colour("grey-4");
+  background-color: govuk-colour("light-grey");
```
Comment (zuzak) [inline src/styles/app.scss:60]: This fails the WCAG 2.1 contrast guidelines (https://webaim.org/resources/contrastchecker/).
(The old colour did too, mind)

**Review by zuzak — COMMENTED** [review]
The [GOV.UK Design System release-notes](https://github.com/alphagov/govuk-frontend/releases/tag/v3.0.0) for v3.0.0 mention a few things that this commit doesn't fix. For example, we use the `data-module` attribute for our accordions: if we're using the GDS code for that, we'll need to update it here too.

I remember vaguely looking into this ticket a while ago and putting it on the backburner because there were lots of small changes that I didn't have time for.

**Comment by parth-ho** [comment]
@zuzak I'm not sure how much we should rely on the release notes for this upgrade, that's just what they've broadcasted, and we don't know how comprehensive these notes are. We also have lots of react and custom JS code interacting with the components too. Might be better and more practical to run some comprehensive tests. If all tests still work then I would be confident that everything is ok.

**Comment by zuzak** [comment]
The GDS release notes tend to be comprehensive for each release, but are of course are targeted at more vanilla users of the Design System than us. Looks like that data-module stuff is in our accordions, but as a result of blind copy/pasting -- it doesn't seem to actually do anything.

We should also add a ticket to the backlog to go through the changes from 2.9 to present to see if there are new features we should in turn be implementing.

File: package.json [hunk 2]
```diff
@@ -45,7 +45,7 @@
         "cookie-parser": "^1.4.4",
         "csurf": "^1.10.0",
         "express": "^4.16.4",
-        "govuk-frontend": "2.9.0",
+        "govuk-frontend": "3.10.0",
```
Comment (zuzak) [inline package.json:48]: Wonder why this doesn't use caret notation.

**Review by zuzak — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (LiamMacP) [reply package.json:48]: They do seem to use patch versions, might be wise to change this next time we go into this file.

**Pull request merged**